docs/extends.md

```diff
@@ -233,7 +237,8 @@ manually keep both environments in sync.
 ### Reference
 
 You can use `extends` on any service together with other configuration keys. It
-always expects a dictionary that should always contain the key: `service` and optionally the `file` key.
+expects either a dictionary that should always contain the key: `service` and
```

Comment:
Try not to use `simply` in docs, it can be very frustrating for those new to this where nothing feels simple!

How about something like this maybe:

```
The `extends` key can also take a string, whose value is the name of a `service` defined in the same file.
```